### DISCUSSION TITLE
Vscode "Error : source file not found : .... "

### DISCUSSION THREAD

**Discussion by bukanwibu-scx** [deaafrizal/forum#98] [Troubleshooting Room]
Ada yang pernah ngalamin hal serupa ga? atau tau solusinya kayak gimana. mohon bimbingannya aku dari kemarin struggle banget nyari masalahnya ini dimana bahkan udah nanya temen, kating, dan dosen masih nihil solusi. 
![Screenshot 2024-09-24 170156](https://github.com/user-attachments/assets/73baad29-368d-49eb-bae0-ce5451707e3b)
udah nyari nyari di youtube gpt dll nyoba berbagai cara dibimbing kating juga masih gatau problemnya dimana. Mungkin ada yang tau boleh minta tolong di bimbing ga urgent banget buat nugas deadline nya besok lagi :(

**Comment by Juniardo150604** [comment]
BANG MAU TANYAK KALO BUAT APLIKASI DAFTAR TAMU LEBIH SIMPELNYA PAKE (Raravel atau Flutter)
